tests/two-same-file/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load(
+    "@rules_haskell//haskell:defs.bzl",
+    "haskell_library",
+    "haskell_test",
+)
+
+package(
+    default_testonly = 1,
+    default_visibility = ["//visibility:public"],
+)
+
```

Comment:
```suggestion
```
This seems redundant. Both are `haskell_test` so they should already be `testonly = 1`, and nothing seems to depend on these targets, so they don't need to be `//visibillity:public`.